apis/r/R/TileDBObject.R

```diff
@@ -149,6 +153,20 @@ TileDBObject <- R6::R6Class(
     uri = function(value) {
       if (missing(value)) return(private$tiledb_uri$uri)
       stop(sprintf("'%s' is a read-only field.", "uri"), call. = FALSE)
+    },
+    #' @field .tiledb_timestamp_range Time range for libtiledbsoma
+    #'
+    .tiledb_timestamp_range = function(value) {
```

Comment:
I thought it was going to be 'not dots for active bindings' but 'dots for private elements' ?  

I am fine with either way.

Reply:
@johnkerl requested the leading dot to indicate that while this is a public binding, it's not designed for use by the average user (they should be using `$tiledb_timestamp` instead)

Reply:
Sure but can't win'em all.  Now you have element in the .Rd file that starts with a dot  :man_shrugging: